main.js

```diff
@@ -1,63 +1,101 @@
+		resultsPage = 0;
+		fetchArticles().then(() => {
+			displaySearchResults();
+		});
+	});
+
+	submitButton.addEventListener('click', event => {
+		event.preventDefault();
+		resultsPage = 0;
+		sortSelect.value = 'relevance';
+		fetchArticles().then(() => {
+			displaySearchResults();
+		});
+	});
+	
+	filtersButton.addEventListener('click', event => {
+		event.preventDefault();
+		toggleFilterMenuVisibility();
+	});
+
+	previousPageButton.addEventListener('click', () => {
+		resultsPage--;
+		fetchArticles().then(() => {
+			displaySearchResults();
+		});
+		scroll(0, 0);
+	});
+
+	nextPageButton.addEventListener('click', () => {
+		resultsPage++;
+		fetchArticles().then(() => {
+			displaySearchResults();
+		});
+		scroll(0, 0);
+	});
+}
+
+function displaySearchResults() {
+	const searchResultsDiv = document.getElementById('search-results-container');
+  const articlesDiv = document.getElementById('articles-container');
+	
+	searchResultsDiv.style.display = '';
   
-  while (searchResults.firstChild) {
-    searchResults.removeChild(searchResults.firstChild);
+  while (articlesDiv.firstChild) {
+    articlesDiv.removeChild(articlesDiv.firstChild);
   }
+	
+  const totalHits = articles.response.meta.hits;
+  const totalHitsPara = document.getElementById('total-hits-msg');
+  totalHitsPara.textContent = `Your query returned ${totalHits} hits.`;
+  totalHitsPara.style.display = 'block';
 
-  const articlesOnPage = articles.response.docs;
+  if (totalHits > 0) {
+    const currentPageArticles = articles.response.docs;
 
-  if (articlesOnPage.length > 0) {
-    articlesOnPage.forEach(article => {
-      const articleDiv = document.createElement('div');
-      articleDiv.setAttribute('class', 'article');
+    currentPageArticles.forEach(article => {
+      const articleContainer = document.createElement('article');
 
       const anchor = document.createElement('a');
       anchor.href = article.web_url;
-      articleDiv.appendChild(anchor);
+      anchor.target = '_blank';
+      anchor.setAttribute('class', 'headline-link');
+      articleContainer.appendChild(anchor);
 
       const headline = document.createElement('h2');
       headline.textContent = article.headline.main;
       anchor.appendChild(headline);
 
       const abstractPara = document.createElement('p');
       abstractPara.textContent = article.abstract;
-      articleDiv.appendChild(abstractPara);
+      abstractPara.setAttribute('class', 'article-abstract');
+      articleContainer.appendChild(abstractPara);
 
       const articleImage = article.multimedia.find(image => image.subtype === 'blog225');
-      
       if (articleImage) {
         const imgEl = document.createElement('img');
         imgEl.src = `http://www.nytimes.com/${articleImage.url}`;
-        articleDiv.appendChild(imgEl);
+        imgEl.setAttribute('class', 'article-img');
+        articleContainer.appendChild(imgEl);
       }
 
       const keywordsPara = document.createElement('p');
```

Comment:
in the browser: I think there is a space missing between the first keyword and the `:`